### DISCUSSION TITLE
Discussions đây nhóe guysss

### DISCUSSION THREAD

**Discussion by HoangBao04** [HoangBao04/10A12-Nhom05#5] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by HoangBao04** [comment]
eyyyyyyy

**Comment by HuuNghia10A12PhuNhuan** [comment]
Finally, YESSSSSSSSSSSSS...

**Comment by DuyHieu14-10A12** [comment]
wow cuối cùng

**Comment by DuyHieu14-10A12** [comment]
khi nào deadline vậy mọi ngườii

**Reply by HuuNghia10A12PhuNhuan** [reply]
> khi nào deadline vậy mọi ngườii

Chắc cuối tuần này, tại t nhớ thầy nói là đầu mỗi tuần sẽ có BT mới.

**Comment by HuuNghia10A12PhuNhuan** [comment]
**_Mọi người nhớ tranh thủ thời gian để làm bài nha !!!!_**

**Comment by HuuNghia10A12PhuNhuan** [comment]
![image](https://user-images.githubusercontent.com/92022929/137748066-32c0eae1-809d-4945-9b18-23cb7e063fda.png)
_**Sorry for stressing everyone out**_   **:(((**
